configure

```diff
@@ -527,8 +527,11 @@ def configure_node(o):
   o['variables']['clang'] = 1 if is_clang else 0
 
   if not is_clang and cc_version != 0:
-    o['variables']['gcc_version'] = 10 * cc_version[0] + cc_version[1]
-
+    try:
+        o['variables']['gcc_version'] = 10 * cc_version[0] + cc_version[1]
+    except IndexError:
```

Comment:
I agree that Python encourages [EAFP: Easier to ask for forgiveness than permission.](https://docs.python.org/2/glossary.html#term-eafp). But I wanted to make sure that the code is easily understandable. Anyway I am not going to hold this PR with this discussion. Please continue.